keyboards/wolf/frogpad/info.json

```diff
@@ -0,0 +1,63 @@
+{
+    "manufacturer": "Geon",
+    "keyboard_name": "Frog Pad",
+    "maintainer": "ToastyStoemp",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true
```

Comment:
Since encoders are enabled by default, some sort of encoder functionality should exist at the keyboard level in a `frogpad.c` file, that way it works for QMK Configurator users.
eg.
```c
bool encoder_update_kb(uint8_t index, bool clockwise) {
    if (!encoder_update_user(index, clockwise)) { return false; }
    if (index == 0) {
        if (clockwise) {
            tap_code_delay(KC_VOLU, 10);
        } else {
            tap_code_delay(KC_VOLD, 10);
        }
    } else if (index == 1) {
        if (clockwise) {
            tap_code(KC_PGDN);
        } else {
            tap_code(KC_PGUP);
        }
    }
    return true;
}
```